devito/ops/operator.py

```diff
@@ -30,10 +30,14 @@ def __init__(self, *args, **kwargs):
     def _specialize_iet(self, iet, **kwargs):
         warning("The OPS backend is still work-in-progress")
 
-        # If there is no iteration tree, then there is no loop to be optimized using OPS.
         iteration_tree = retrieve_iteration_tree(iet, mode='normal')
-        if not len(iteration_tree):
+        affine_trees = find_affine_trees(iet).items()
+
+        # If there is no iteration trees nor affine trees,
+        # then there is no loop to be optimized using OPS.
+        if not len(iteration_tree) or not len(affine_trees):
```

Comment:
you don't need `len`

Reply:
in fact, I think this condition should just be

```
if not affine_trees: 
     ....
```

Reply:
ok, fixing